plugins/org.jkiss.dbeaver.ext.db2/src/org/jkiss/dbeaver/ext/db2/manager/DB2TableColumnManager.java

```diff
@@ -73,7 +73,7 @@
     @Override
     public boolean canEditObject(DB2TableColumn object)
     {

```

Comment:
Consider moving `{` onto the previous line for all method declarations according to the code style